atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -288,26 +265,15 @@ protected CassandraKeyValueServiceImpl(Logger log,
         this.cassandraTables = new CassandraTables(clientPool, configManager);
     }
 
-    private LockLeader whoIsTheLockCreator() {
-        return leaderConfig
-                .map((config) -> config.whoIsTheLockLeader())
-                .orElse(LockLeader.I_AM_THE_LOCK_LEADER);
-    }
-
-    @Override
-    public synchronized void cleanUpOnInitFailure() {
-        // no-op
-    }
-
-    @Override
     public boolean isInitialized() {
-        return isInitialized;
+        return wrapper.isInitialized();
     }
 
-    @Override
-    public synchronized void tryInitialize() {
-        assertNotInitialized();
+    protected void initialize(boolean asyncInitialize) {
```

Comment:
looks unused?

Reply:
It's used in the `CassandraExpiringKeyValueService`, which extends CKVS.